nservicebus/messaging/delayed-delivery.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Delayed Delivery
+summary: Delay delivery of messages until a later time.
+tags:
+- Defer
+---
+
+When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered using `bus.Defer(...)` instead of `bus.Send(...)`.
```

Comment:
`bus.` is not really correct in v6. perhaps  `Defer(...)` instead of `Send(...)`.